CRP.Mvc/Services/SlothService.cs

```diff
@@ -68,8 +69,20 @@ public async Task<CreateSlothTransactionResponse> CreateTransaction(CreateTransa
                 var url = "transactions";
 
                 var response = await client.PostAsJsonAsync(url, transaction);
-                var result = await response.GetContentOrNullAsync<CreateSlothTransactionResponse>();
-                return result;
+                try
+                {
+                    var result = await response.GetContentOrNullAsync<CreateSlothTransactionResponse>();
+                    return result;
```

Comment:
There is no reason to save the result to immediately return it, just return the await